packages/flutter_tools/lib/src/asset.dart

```diff
@@ -683,6 +683,10 @@ void _parseAssetsFromFolder(
   for (final FileSystemEntity entity in lister) {
     if (entity is File) {
       final String relativePath = globals.fs.path.relative(entity.path, from: assetBase);
+      if (_isFileHidden(relativePath)) {
+        globals.printStatus("File '$relativePath' is hidden and won't be included in the bundle.");
```

Comment:
nit: don't log things that aren't actionable

Reply:
I thought that could be an useful information to the user. Should I then remove this completely or transform it to more actionable version like 'X is not included. Unhide it to inlclue'?

Reply:
In that case, I would add this as a printTrace so that it only shows up in verbose mode. The regular printStatus is reserved for highly actionable or important messages